src/auth/signup-form/signup-form.js

```diff
@@ -1,34 +1,157 @@
 import React, { Component } from "react";
 import { Form, Button } from "react-bootstrap";
+import {TextField} from '@material-ui/core'
 import "./signup-form.scss";
+import Grid from '@material-ui/core/Grid';
+import { makeStyles} from '@material-ui/core/styles';
+
 
 class SignUpForm extends Component {
+
+  constructor(){
+    super();
+    this.state={
+        first_name:'',
+        last_name:'',
+        email:'',
+        phone_no:'',
+        password:'',
+        cnf_password:''
+    }
+
+    this.onSubmit= this.onSubmit.bind(this)
+}
+
+handleChange = (event, params) => {
+  event.preventDefault();
+   if (params === "first_name")
```

Comment:
It's not a good way to handle change use a global name attribute on textarea and then use this.setState({ [e.target.name]: e.target.value });

Reply:
ok , made the changes please review